clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -264,6 +267,13 @@ public synchronized long lastSuccessfulUpdate() {
     }
 
     /**
+     * The max allowable age of metadata.
+     */
+    public long maxAge() { 
+        return this.metadataExpireMs; 
```

Comment:
Why are we using two different names here and elsewhere? It would be good to be consistent. Also, whatever we choose, we generally include the unit at the end (e.g. `maxAgeMs`).